crates/rspack_macros/src/merge.rs

```diff
@@ -86,8 +86,14 @@ fn body(data: Data, chose_base: Option<&Attribute>) -> Result<TokenStream> {
               .enumerate()
               .map(|(i, f)| {
                 let field_name = &f.ident;
-                let a_name = Ident::new(&format!("a_{i}"), field_name.span());
-                let b_name = Ident::new(&format!("b_{i}"), field_name.span());
+                let a_name = Ident::new(
+                  &format!("a_{}", itoa::Buffer::new().format(i)),
```

Comment:
This is inside macro, which runs in build time, not affect to performance, let's keep it as is